### DISCUSSION TITLE
FaceToEdge() for DFaust

### DISCUSSION THREAD

**Discussion by koustav123** [pyg-team/pytorch_geometric#2970] [Q&A] [answered]
I am trying to create the DFaust dataset with the FaceToEdge pre_transform. I get an assertion error.

```
train_dataset = DynamicFAUST(root='/data2/ghosalk/DFAUST/', subjects=["50002", "50004", "50007", "50009", "50020"], pre_transform=T.FaceToEdge())
test_dataset = DynamicFAUST(root='/data2/ghosalk/DFAUST/', subjects=["50021", "50022", "50025", "50026", "50027"], pre_transform=T.FaceToEdge())
```

![image](https://user-images.githubusercontent.com/3684879/129367801-5c8f7a38-1674-412c-ba80-07feb5a2be4c.png)

I will appreciate any pointers. Thanks in advance

**Comment by rusty1s** [comment, marked as answer]
Thanks for reporting. This is now fixed in master, see https://github.com/rusty1s/pytorch_geometric/commit/8e29fd2d68fcb89487e42f95e2fc96a0201650c1.
